packages/server/graphql/types/RetrospectiveMeeting.ts

```diff
@@ -58,6 +58,10 @@ const RetrospectiveMeeting: GraphQLObjectType<any, GQLContext> = new GraphQLObje
       type: new GraphQLList(new GraphQLNonNull(AutogroupReflectionGroup)),
       description: 'The suggested reflection groups created by OpenAI'
     },
+    resetReflectionGroups: {
+      type: new GraphQLList(new GraphQLNonNull(AutogroupReflectionGroup)),
```

Comment:
Using the `AutogroupReflectionGroup` type for both `autogroupReflectionGroups` and `resetReflectionGroups` could be confusing. I'm open to other names